samples/subsys/console/echo/README.rst

```diff
@@ -0,0 +1,43 @@
+.. zephyr:code-sample:: console_echo
+   :name: Console echo
+   :relevant-api: console_api
+
+   Echo an input character back to the output using the Console API.
+
+Overview
+********
+
+This example shows how the :c:func:`console_getchar` and
+:c:func:`console_putchar` functions can be used to echo an input character
+back to the console.
+
+Requirements
+************
+
+UART console is required to run this sample.
```

Comment:
@kartben I saw you mentioned this should change - I am perhaps referring mistakenly to the terminal program I am using here (picocom). This is also consistent with the other console samples.

Reply:
Ya I removed my comment as I was mostly getting confused between shell and console :) I don't think a console that's *not* UART makes sense so nevermind :)